yt/frontends/chimera/data_structures.py

```diff
@@ -0,0 +1,309 @@
+"""
+Chimera data structures
+
+
+
+"""
+
+import os
+
+import numpy as np
+
+from yt.geometry.geometry_handler import YTDataChunk
+from yt.geometry.unstructured_mesh_handler import UnstructuredIndex
+from yt.utilities.file_handler import HDF5FileHandler
+from yt.utilities.io_handler import io_registry
+from yt.utilities.on_demand_imports import _h5py as h5py
+
+try:
+    from yt.data_objects.unstructured_mesh import SemiStructuredMesh
+except ModuleNotFoundError:
+    from yt.data_objects.index_subobjects.unstructured_mesh import SemiStructuredMesh
```

Comment:
I guess this isn't needed anymore ?
```suggestion
from yt.data_objects.index_subobjects.unstructured_mesh import SemiStructuredMesh
```